R/scope.R

```diff
@@ -375,3 +373,15 @@ renv_scope_umask <- function(umask, envir = parent.frame()) {
   defer(Sys.umask(oldmask), envir = envir)
   invisible(oldmask)
 }
+
+renv_scope_setwd <- function(dir, envir = parent.frame()) {
+  owd <- setwd(dir)
+  defer(setwd(owd), envir = envir)
+  invisible(owd)
+}
+
+renv_scope_sandbox <- function(envir = parent.frame()) {
```

Comment:
See also `renv_scoped_sandbox()` (which has a typo in the name, oops) in `test-sandbox.R`

Reply:
Thanks! I retooled those tests.